components/SkipLogin/SkipLoginButton.tsx

```diff
@@ -1,14 +1,14 @@
 import styles from "./SkipLoginButton.module.css";
 
 function SkipLoginButton() {
-  function handleClick(e) {
+  function handleClick() {
     alert("This feature will be available in a later version of the app.");
   }
 
   return (
-    <a href="#" onClick={handleClick}>
-      <button className={styles.skip}>Skip Login and just Browse</button>
-    </a>
+    <button onClick={handleClick} className={styles.skip}>
+      Skip Login and just Browse
```

Comment:
```suggestion
      skip login and just browse
```

english is mostly small caps. Another variation would be: Skip login and just browse 
😊